src/sidebar/components/annotation-action-button.js

```diff
@@ -6,18 +6,21 @@ const { createElement } = require('preact');
 
 const SvgIcon = require('./svg-icon');
 
+/**
+ * A simple icon-only button for actions applicable to annotations
+ */
 function AnnotationActionButton({
-  className = '',
   icon,
-  isDisabled,
+  isActive = false,
   label,
-  onClick,
+  onClick = () => null,
 }) {
   return (
     <button
-      className={classnames('annotation-action-button', className)}
+      className={classnames('annotation-action-button', {
+        'annotation-action-button--active': isActive,
```

Comment:
IIRC we discussed using `is-<state>` class naming for dynamically-added state classes recently, per the conventions in our [CSS guide](https://github.com/hypothesis/frontend-toolkit/blob/master/docs/css-style-guide.md#naming).

Reply:
Ah, yes, this fell into a bit of a grey area in my brain, but you're right. I'll update.